### PR TITLE
Dev charts for JupyterHub

### PR DESCRIPTION
**Pull request by jireva**
Currently does not have access to any data.

### CODE REVIEW

File: dev/data/config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+singleuser:
+  image:
+    # You should replace the "latest" tag with a fixed version from:
```
Comment (bodymindarts) [inline dev/data/config.yaml:3]: Is this comment required?

File: dev/README.md [hunk 2]
```diff
@@ -29,4 +29,11 @@ $ curl 'localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept:
 {"data":{"userLogin":{"authToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI2MTc2YmQ2NmQ0MmFkYWIzNjM2MmEyY2QiLCJuZXR3b3JrIjoibWFpbm5ldCIsImlhdCI6MTYzNTE3MTY4Nn0.n-p5sA9meAmZrVOdngYr216jG3LKOFsFdJmVw6XND3A"}}}
 ```
 
+To test the jupyter lab:
```
Comment (bodymindarts) [inline dev/README.md:32]: Can you move this to its own subheading lower in the README.

**Review by bodymindarts — COMMENTED** [review]
As we are not actually creating a chart for this it won't be necessary to add a TestFlight step. We can add it to staging directly in the deployments repo once we have the `addons` lifecycle.

File: dev/data/config.yaml (hunk 1 above)
Comment (jireva) [reply dev/data/config.yaml]: Removed it.

File: dev/README.md (hunk 2 above)
Comment (jireva) [reply dev/README.md:36]: Done.

**Review by bodymindarts — DISMISSED** [review]

**Pull request closed without merging**